gcp_variant_transforms/libs/annotation/vep/vep_runner.py

```diff
@@ -228,12 +233,12 @@ def wait_until_done(self):
 
   def _is_done(self, operation):
     # type: (str) -> bool
-    # TODO(bashir2): Catch exceptions in the following call and also silence the
-    # log messages of googleapiclient.discovery module for the next call of the
-    # API since they flood the log file.
+    # TODO(bashir2): Silence the log messages of googleapiclient.discovery
+    # module for the next call of the API since they flood the log file.
     # pylint: disable=no-member
-    request = self._pipeline_service.projects().operations().get(name=operation)
-    is_done = request.execute()['done']
+    request = self._pipeline_service.projects().operations().get(
+        name=operation)
+    is_done = request.execute(num_retries=_NUMBER_OF_API_CALL_RETRIES)['done']
```

Comment:
Curious: were you actually seeing errors in this case that you needed retry? Asking b/c we don't do this for our integration tests and it seems like it has been fine so far, but I wonder whether we should add it if you are seeing errors in v2.

Reply:
I am not sure if this is a V2 specific issue (vs a general possibility when talking to external services) but I saw it exactly here when we are polling whether the operations are done. That experience was particularly painful because I was running it on some big VCFs and it happened after a few hours (crashing VT). The exception message was something indicating a server connection error.

Re. integration tests: Given the high availability of our services, I don't expect that we see issues like this very often. In case of here and the polling because we keep doing this every 30 seconds for a few hours, it is more likely to happen.